packages/atlas/src/components/_comments/ReactionPopover/ReactionPopover.tsx

```diff
@@ -40,13 +40,14 @@ const getTranslateNumber = (idx: number) => {
 export const ReactionPopover: React.FC<ReactionPopoverProps> = ({ onReactionClick, disabled }) => {
   const [isOpen, setIsOpen] = useState(false)
   const smMatch = useMediaMatch('sm')
+  const reactionPopoverDismissed = usePersonalDataStore((state) => state.reactionPopoverDismissed)
   const reactions = Object.entries(REACTION_TYPE).map(([key, value]) => ({
-    name: key as ReactionType,
-    value,
+    reactionId: Number(key) as ReactionId,
```

Comment:
Why does `reactionId` need to be a number?

Reply:
The QN also returns numbers in the `reactionId` field. `Object.entries` convert keys into a string, that's why I'm changing this to the number.  I want to make sure that type is the same as `ReactionId`.

Reply:
That's a bit weird since it accepts string as an input

Reply:
Nvm, it's a number as wel